packages/react/src/components/SelectUsersModal/SelectUsersModal.jsx

```diff
@@ -125,16 +126,26 @@ const mapUsers = (
     };
   });
 
+const flattenUsers = (results, user) => {
+  return user.children && user.children.length > 0
+    ? user.children.reduce(flattenUsers, results)
+    : results.concat(user);
+};
+
 const SelectUsersModal = ({ isOpen, onClose, onSubmit, users, initialSelectedUsers, i18n }) => {
   const mergedI18n = { ...defaultProps.i18n, ...i18n };
 
   const [selectedUsers, setSelectedUsers] = useState(initialSelectedUsers);
 
+  const canSaveRef = useRef(false);
```

Comment:
Added this ref to handle the disabling of the "OK" button. Previous, it wasn't possible to remove a user and save the new empty list.